### PR TITLE
Comments and likes

### PR DESCRIPTION
**Pull request by poornimagodavarthy2**
Created a textbox to put in comments and made a comment component on the frontend

### CODE REVIEW

File: packages/react-frontend/src/components/Comment.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    margin-bottom: 5px;
+    font-family: 'League Spartan', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+  }
+  
+  .comment-text {
+    color: white;
+    
+  }
```
Comment (ishwaryanam) [inline packages/react-frontend/src/components/Comment.css:38]: i like how you organized your css all of your component names are descriptive!

File: packages/react-frontend/src/components/Comments.jsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+function Comments({ playlistId, userId }) { 
+  const [newComment, setNewComment] = useState("");
+
+  const handleCommentChange = (event) => {
+    setNewComment(event.target.value);
+  };
+  
+  const submitComment = async (commentText) => {
+    try {
+      const response = await fetch(`http://localhost:8000/playlists/${playlistId}/comments`, {
```
Comment (ishwaryanam) [inline packages/react-frontend/src/components/Comments.jsx:14]: linking the frontend and backend looks good!

File: packages/react-frontend/src/pages/Playlist.jsx [hunk 3]
```diff
@@ -37,62 +52,70 @@ function Playlist() {
...
+          Add Song
+        </button>
+      </div>
     );
-
-    
   }
 
+  const toggleComments = () => {
+    setShowComments(!showComments);
```
Comment (ishwaryanam) [inline packages/react-frontend/src/pages/Playlist.jsx:106]: looks good!

**Review by ishwaryanam — COMMENTED** [review]
your code looks good to me! i like how organized everything is good job!

**Review by hahnspena102 — APPROVED** [review]
LGTM. Overall looks good to me! I would suggest changing the component names to "CommentBox" and "CommentMap" but that's just a preference thing.

File: packages/react-frontend/src/components/Comment.css (hunk 1 above)
Comment (hahnspena102) [reply packages/react-frontend/src/components/Comment.css:38]: i agree, very easy to read!

File: packages/react-frontend/src/components/Comment.jsx [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+  return (
+    <div className="comment">
+      <div className="comment-profile-circle"></div>
+      <div className="comment-content">
+        <div className="comment-username">{username}</div>
+        <div className="comment-text">{commentText}</div>
+      </div>
+    </div>
+  );
+};
```
Comment (ArjunR27) [inline packages/react-frontend/src/components/Comment.jsx:14]: This looks good to me!

File: packages/react-frontend/src/pages/Playlist.jsx [hunk 5]
```diff
@@ -37,62 +52,70 @@ function Playlist() {
...
-              .then(json => {
-                setPlaylist(json["playlist_list"]);
-              })
-              .catch(error => {
-                console.log(error);
-              });
+      console.log("Adding song:", song);
+
+      fetch("http://localhost:8000/playlists/" + path, {
+        method: "POST",
```
Comment (ArjunR27) [inline packages/react-frontend/src/pages/Playlist.jsx:65]: We are going to have to change this, now that CI/CD is up and running. But this is a minor fix just need to change it to "https://songsmith.azurewebsites.net/playlists" + path

**Pull request merged**